ballerina/http_client_connection_pool.bal

```diff
@@ -27,11 +27,13 @@ configurable int maxActiveStreamsPerConnection = 100;
 # + maxIdleConnections - Maximum number of idle connections allowed per pool.
 # + waitTime - Maximum amount of time (in seconds), the client should wait for an idle connection before it sends an error when the pool is exhausted
 # + maxActiveStreamsPerConnection - Maximum active streams per connection. This only applies to HTTP/2. Default value is 100
+# + evictableIdleTime - Evictable time for an idle connection in seconds. Default value is 5 minutes
 public type PoolConfiguration record {|
     int maxActiveConnections = maxActiveConnections;
     int maxIdleConnections = maxIdleConnections;
     decimal waitTime = waitTime;
     int maxActiveStreamsPerConnection = maxActiveStreamsPerConnection;
+    int evictableIdleTime = 5 * 60;
```

Comment:
We can directly put 300 here right?
```suggestion
    int evictableIdleTime = 300;
```

Reply:
Yeah. Felt that this would be easier for a user to identify its 5 minutes

Reply:
Yes, that's fine. But since we are anyway adding the default value in the doc comment, I don't think we need to give the value like this

Reply:
ack